src/viewer/index.ts

```diff
@@ -297,7 +297,7 @@ export class Viewer {
         return out;
     }
 
-    loadStructureFromUrl<P, S>(url: string, format: BuiltInTrajectoryFormat, isBinary: boolean, config?: {props?: PresetProps; matrix?: Mat4; reprProvider?: TrajectoryHierarchyPresetProvider<P, S>, params?: P}) {
+    loadStructureFromUrl<P, S>(url: string, format: BuiltInTrajectoryFormat, isBinary: boolean, config?: {props?: PresetProps & { structureLabel?: string }; matrix?: Mat4; reprProvider?: TrajectoryHierarchyPresetProvider<P, S>, params?: P}) {
```

Comment:
There's `dataLabel` available for `loadStructureFromData`, could consider naming this the same, but either way seems fine to me.

Reply:
Does `dataLabel` serve similar purpose?

Reply:
Both get passed to the builder in the same fashion: https://github.com/molstar/rcsb-molstar/blob/db1347de0f549634a94d962fee346f82561934da/src/viewer/helpers/model.ts#L29

Though I don't think we use that functionality.